docs/python/python-tutorial.md

```diff
@@ -227,24 +227,24 @@ plt.show()                   # Display the plot
 
 Next, try running the file in the debugger using the "Python: Current file" configuration as described in the last section. (If you still have `"stopOnEntry": true` in that configuration, you need to select the run command again to continue.)
 
-Unless you're using an Anaconda distribution or have previously installed the matplotlib package, you should see the message, "ModuleNotFoundError: No module named 'matplotlib'". Such a message indicates that the required package isn't available in your system.
+Unless you're using an Anaconda distribution or have previously installed the `matplotlib` package, you should see the message, "ModuleNotFoundError: No module named 'matplotlib'". Such a message indicates that the required package isn't available in your system.
 
-To install the matplotlib package (which also installs numpy), stop the debugger and run **Python: Create Terminal** from the Command Palette, which opens a command prompt for your selected interpreter. Then enter the following commands as appropriate for your operating system (commands may require elevation if the Python interpreter is installed in a protected area of the file system):
+To install the `matplotlib` package (which also installs `numpy` as a dependency), stop the debugger and run **Python: Create Terminal** from the Command Palette. This will open a command prompt for your selected interpreter. Then enter the following commands as appropriate for your operating system (commands may require elevation if the Python interpreter is installed in a protected area of the file system):
 
 > **Note**: If you are unable to install the package, please [file an issue on GitHub](https://github.com/Microsoft/vscode-docs/issues) so we can help you investigate.
 
 ```bash
-# pip[3] install commands may require elevation
+# pip install commands may require elevation
 
 # macOS
-pip3 install matplotlib
+python -m pip install matplotlib
```

Comment:
It's fine to use `python3`, but `pythonX.Y -m pip` is preferred over `pip`. This is because 

- `pip` may not be on the `PATH` even if `python` is **OR**
- The version of `pip` on the `PATH` may not be the same `pip` corresponding to the "current" Python

See https://bugs.python.org/issue22295 for a discussion of why `python -m pip` over `pip` (this is from the maintainers of `pip`)

Other links:

- https://docs.python.org/3/installing/index.html#basic-usage
- https://www.reddit.com/r/learnpython/comments/42h5j0/which_is_better_python_m_pip_install_or_pip/

Reply:
OK, thanks for the clarification. On MacOS, then, would it still be python3 -m pip rather than python -m pip?

Reply:
I think it's up to you. There'd be nothing wrong with `python3` (especially given [PEP 394][1]) but I'd also imagine 99.9% of readers will interpret `python` to mean "the `python` executable that you are using" (even for users that don't know what a `PATH` is).

[1]: https://www.python.org/dev/peps/pep-0394/

Reply:
Thanks. For the time being, I'll trust your judgment on this and see if any other feedback comes in otherwise. Thanks for the discussion.

Reply:
Thanks to you as well!